packages/@angular/cli/commands/completion.ts

```diff
@@ -117,14 +117,15 @@ const CompletionCommand = Command.extend({
         });
       }
 
-      let opts = '';
+      let opts: string[] = [];
       if (command.blueprints && command.blueprints[0]) {
-        opts += extractBlueprints(command.blueprints);
+        opts = opts.concat(extractBlueprints(command.blueprints));
       }
 
       if (command.availableOptions && command.availableOptions[0]) {
-        opts += extractOptions(command.availableOptions);
-        caseBlock = caseBlock + '    ' + com.sort().join('|') + ') opts="' + opts + '" ;;\n';
+        opts = opts.concat(extractOptions(command.availableOptions));
+        const optsStr = opts.sort().join(' ');
+        caseBlock = caseBlock + '    ' + com.sort().join('|') + ') opts="' + optsStr + '" ;;\n';
```

Comment:
template string: sure.

I dont know which parantesis you mean.

The target String looks like this:
```
case $words[cword] in
          ng|help) opts="foo bar --baz" ;;
```

There is only a right paranthesis in case statements